src/constants.lua

```diff
@@ -11,6 +11,10 @@ constants.oneYearMs = 31536000 * 1000
 
 -- EPOCHS
 constants.defaultEpochDurationMs = constants.oneDayMs
+constants.decayStartingRewardRate = 0.001
+constants.defaultRewardRate = 0.0005
+constants.rewardDecayStartEpoch = 365
+constants.rewardDecayEndEpoch = 548
```

Comment:
`End` in the name is squishy because it's not clear whether it's inclusive or exclusive. Perhaps `Last` is clearer (and uses an inclusive value)?

Reply:
[0d8533f](https://github.com/ar-io/ar-io-network-process/pull/272/commits/0d8533f23cc03f72ce61f34f1e41319b1b7ef2bd)